include/triton/Conversion/TritonGPUToLLVM/Utility.h

```diff
@@ -31,243 +30,247 @@ using namespace mlir;
 using namespace mlir::triton;
 
 namespace mlir::triton {
+
+// Returns CTA level thread idx
+inline Value getThreadId(OpBuilder &rewriter, Location loc) {
+  Value tid =
+      rewriter.create<::mlir::gpu::ThreadIdOp>(loc, ::mlir::gpu::Dimension::x);
+  Type i32_ty = rewriter.getIntegerType(32);
+  return rewriter.create<arith::IndexCastOp>(loc, i32_ty, tid);
+}
+
 struct TritonLLVMOpBuilder {
-  TritonLLVMOpBuilder(const Location &loc, RewriterBase &builder)
+  TritonLLVMOpBuilder(const Location &loc, OpBuilder *builder)
```

Comment:
You could have kept this as `OpBuilder &builder` and just taken the address in the constructor, but I guess it doesn't really matter.

Reply:
Good idea - I'll change it back tomorrow (just don't merge before I get a chance).

Reply:
done